src/sql.rs

```diff
@@ -242,6 +242,22 @@ impl Sql {
             }
         }
 
+        // Reset some options if IsChatmail is true,
+        // because there were reports from users who had these options set to something else,
+        // and then couldn't change them because they are hidden in the UI for chatmail profiles
+        if update_email_configs && context.get_config_bool(Config::IsChatmail).await? {
+            // The default for MvboxMove is actually "1", and it's usually set to "0" in `configure.rs`.
+            context
+                .set_config_internal(Config::MvboxMove, Some("0"))
+                .await?;
+            context
+                .set_config_internal(Config::OnlyFetchMvbox, None)
+                .await?;
+            context
+                .set_config_internal(Config::ShowEmails, None)
```

Comment:
See my comment above, the problem with these high-level APIs is that we may want to get rid of `Config::ShowEmails` constant once we remove it, and then to have migration code we will have to keep it or rewrite the migration. Better write the migration with SQL statements and no high-level APIs.